taichi/codegen/llvm/codegen_llvm.cpp

```diff
@@ -2757,6 +2749,49 @@ llvm::Type *TaskCodeGenLLVM::get_real_func_ret_type(Function *real_func) {
   return llvm::StructType::get(*llvm_context, tps);
 }
 
+void TaskCodeGenLLVM::create_return(llvm::Value *buffer,
+                                    llvm::Type *buffer_type,
```

Comment:
Can we obtain buffer_type from `buffer->getType()`?

Reply:
No... While it works fine on linux machines, it doesn't work on Windows because the pointers are opaque pointers, and `buffer->getType()` only returns `ptr` in LLVM 15. I don't know why it works on linux machines... maybe the LLVM binaries are different....